application/gqlgen/schema.graphql

```diff
@@ -137,6 +137,7 @@ type Post {
     updatedAt: Time!
     url: String
     cost: String
+    kilograms: String
```

Comment:
`String`?  Wouldn't a float or number be more appropriate?

Reply:
When we get to conversions I'd expect to have some decimal numbers, so float might be better for that from the gql schema portion

Reply:
I was mimicking how we did it for `cost`, but I'll make it a float now.